src/GeneralStateTestsFiller/stEWASMTests/logTopicsFiller.yml

```diff
@@ -0,0 +1,139 @@
+# Test log with topics
+logTopics:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    # tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+    # called contract, very simple, just calls log
+    deadbeef00000000000000000000000000000000:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "log" (func $log (param i32 i32 i32 i32 i32 i32 i32)))
+          (memory 1)
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+          (func $main
+            (call $log
+              ;; dataOffset i32ptr the memory offset to load data from (bytes)
+              (i32.const 0)
+              ;; length i32 the data length
+              (i32.const 0x000000ff)
+              ;; numberOfTopics i32 the number of topics following (0 to 4)
+              (i32.const 2)
+              ;; topic1 i32ptr the memory offset to load topic1 from (u256)
+              (i32.const 0)
+              ;; topic2 i32ptr the memory offset to load topic2 from (u256)
+              (i32.const 0)
```

Comment:
All these, including the data above, are pointing to the same empty memory space. It would be better if they would contain different data.